addons/account/i18n/account.pot

```diff
@@ -3812,13 +3812,6 @@ msgstr ""
 msgid "Computed Balance"
 msgstr ""
 
-#. module: account
-#: model:ir.model.fields,help:account.field_account_move_line__tax_audit
-msgid ""
-"Computed field, listing the tax grids impacted by this line, and the amount "
-"it applies to each of them."
-msgstr ""
-
```

Comment:
You don't need to this on any of the `.po` files on master. The way it works on **master** is that each time you have a new version, all the `.po` and `.pot` files are exported again and we create a transiflex project. So all unused fields are automatically removed.